examples/Hermitian_Phasing.py

```diff
@@ -48,34 +48,33 @@
                 * s.exp(+1j * 2 * s.pi * 100 * t2 - abs(t2) * 50 * s.pi)
             ),
             [
-                ("power", nddata(r_[0:4:25j], "power")),
-                ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
-                ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
+                ("power", psd.nddata(r_[0:4:25j], "power")),
+                ("ph1", psd.nddata(r_[0:4] / 4.0, "ph1")),
+                ("t2", psd.nddata(r_[0:0.2:256j] - echo_time, "t2")),
             ],
             {"ph1": 1},
             "power",
             "enhancement",
         ),
     ]:
         fl.basename = "(%s)" % label
-        fig, ax_list = subplots(1, 4, figsize = (7,7))
+        fig, ax_list = subplots(1, 4, figsize=(7, 7))
```

Comment:
Should not be passing flake. JF will look at that.

Reply:
I pushed updated version of the flake8 action, but it's not running b/c you  have conflicts

Reply:
ran flake and fixed all conflicts for this guy